app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -96,7 +97,7 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         )
     }
 
-    @Suppress("ComplexMethod")
+    @Suppress("ComplexMethod", "LongMethod")
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
```

Comment:
This is already an _enormous_ method, and I hate adding to it, but I can see why fixing it might be out of scope here.